src/lib/accessibilityRouting.ts

```
@@ -0,0 +1,110 @@
+  },
+  assistive_listening: {
+    label: "FM/Infrared Assistive Listening Device",
+    description: "Direct audio amplification system for auditorium sightlines.",
+  },
+};
+
+/**
+ * Checks if accommodation type requires direct University Disability Services routing (#4277).
+ */
+export function isCriticalAccommodation(type: string): boolean {
+  return ["asl_interpreter", "braille_materials", "wheelchair_shuttle", "assistive_listening"].includes(type);
```

ESLint

[error] Replace `"asl_interpreter",·"braille_materials",·"wheelchair_shuttle",·"assistive_listening"` with `⏎····"asl_interpreter",⏎····"braille_materials",⏎····"wheelchair_shuttle",⏎····"assistive_listening",⏎··`

(prettier/prettier)